entrypoint.go

```diff
@@ -116,38 +107,43 @@ func UpdateConfigContent(config []byte, replacements []Replacement) []byte {
 
 // BuildReplacementsFromEnv Build []Replacement from env vars
 func BuildReplacementsFromEnv() ([]Replacement, error) {
+	letsEncryptURLs := map[string]string{
+		"staging":    "https://acme-staging.api.letsencrypt.org/directory",
+		"production": "https://acme-v01.api.letsencrypt.org/directory",
+	}
 
 	var configReplacements []Replacement
 
 	envVars := GetEnvVarModels()
 	for _, envvar := range envVars {
 		value := os.Getenv(envvar.Name)
-		if value == "" && envvar.Required {
-			return configReplacements, fmt.Errorf("Missing required env var: %s. Description: %s", envvar.Name, envvar.Desc)
-		}
+		if value == "" {
+			if envvar.Required {
+				return configReplacements, fmt.Errorf("missing required env var: %s. Description: %s", envvar.Name, envvar.Desc)
+			}
 
-		if value != "" {
-			if envvar.Name == "LETS_ENCRYPT_CA" {
-				if value == "staging" {
-					value = "https://acme-staging.api.letsencrypt.org/directory"
-				} else if value == "production" {
-					value = "https://acme-v01.api.letsencrypt.org/directory"
-				}
-			} else if envvar.Name == "SANS" {
-				sans := strings.Split(value, ",")
-				value = ""
-				for _, san := range sans {
-					value += "\"" + san + "\", "
-				}
-				value = strings.TrimRight(value, ", ")
+			if envvar.Default == "" {
+				continue
 			}
-			configReplacements = append(configReplacements, Replacement{
-				Key:   envvar.Name,
-				Value: value,
-			})
-		} else if value == "" && !envvar.Required {
+
 			value = envvar.Default
```

Comment:
@briskt This only changed the local variable, not anything that was returned. My new code doesn't skip adding values that change due to `Default`.

I could keep it the same way (use continue at the end of the first if statement), but then there's no real purpose in `Default`.

Reply:
It did, but only if it was set to `"staging"` and not if it was set to `""`, the latter would of (if it worked properly) set it to `"staging"`, but do no further changes to it.